definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx

```diff
@@ -9,7 +9,79 @@ config {
+        funded_start_declaration: "True if the declaration is in an eligible, payable, or paid state.",
+        retained1_declaration_id: "ID of the participant's retained-1 declaration.",
+        retained1_declaration_state: "State of the participant's retained-1 declaration.",
+        retained1_declaration_date: "Date evidence was submitted for participant's retained-1 declaration.",
+        funded_retained1_declaration: "True if the declaration is in an eligible, payable, or paid state.",
+        retained2_declaration_id: "ID of the participant's retained-2 declaration.",
+        retained2_declaration_state: "State of the participant's retained-2 declaration.",
+        retained2_declaration_date: "Date evidence was submitted for participant's retained-2 declaration.",
+        funded_retained2_declaration: "True if the declaration is in an eligible, payable, or paid state.",
+        completed_declaration_id: "ID of the participant's completed declaration.",
+        completed_declaration_state: "State of the participant's completed declaration.",
+        funded_completed_declaration: "True if the declaration is in an eligible, payable, or paid state.",
+        completed_declaration_date: "Date evidence was submitted for participant's completed declaration.",
+        latest_outcome_state: "State of participant's latest outcome for their NPQ. Possible fields: Passed, Failed, Voided.",
+        latest_outcome_date: "Date of completion for the participant's latest outcome.",
+        count_of_fails: "Counts the number of Failed outcome states for a participant.",
+        cohort_id: "ID of cohort.",
+        npq_lead_provider_id: "ID of Lead Provider.",
+        participant_identity_id: "",
+        primary_establishment: "Records whether the user's establishment at time of registration was a primary establishment or not. Based on GIAS data.",
+        school_ukprn: "UKPRN of the participant's establishment ",
+        teacher_catchment: "The catchment zone the participant works in within the UK i.e. England, Wales, Northern Ireland etc. International applicants recorded as 'another'.",
+        teacher_catchment_country: "The country the participant works in.",
+        teacher_catchment_iso_country_code: "The ISO code for the country the participant works in.",
+        tsf_primary_eligibility: "TRUE/FALSE for whether a school is eligible for TSF (£200) owing to it being a primary establishment.",
+        tsf_primary_plus_eligibility: "TRUE/ FALSE for whether a school is eligible for the base £200 as well as the additional £600 based on whether they are a primary school and have fewer than 150 pupils. Pupil count based on GIAS data at time of registration.",
+        works_in_nursery: "Indicates whether the participant works in a nursery.",
+        works_in_school: "Indicates whether the participant works in a school. ",
+        latest_user_id: ""

```

Comment:
Might need to clarify what this is.

Reply:
If profile_user_id is the ID from the participant_profiles table, which we'd usually define as participant_profile_id, does the CASE statement for this field then need to change?